src/RA_Dlg_GameLibrary.cpp

```diff
@@ -11,7 +11,7 @@
 #include "RA_md5factory.h"
 
 
-inline constexpr auto CALLBACK KEYDOWN(int vk)->decltype((GetAsyncKeyState(vk) & 0x8000) ? true : false)
+inline constexpr bool CALLBACK KEYDOWN(int vk) noexcept
 {
```

Comment:
It can't be auto deduced since it's not simple enough so we either need to put the type or a deduction. Didn't really matter to me so we put the type.